src/hooks/useCombobox/__tests__/getToggleButtonProps.test.js

```diff
@@ -208,6 +208,8 @@ describe('getToggleButtonProps', () => {
         )
 
         fireEvent.click(toggleButton)
+        // fireEvent.blur(input)
```

Comment:
Can we remove this?

Reply:
sure -- comment was just for symmetry with the comments above

Reply:
I mean also the line below, I don't see why is it needed.

Reply:
the `input.blur()` is needed actually (or `toggleButton.focus()` alternatively). i had assumed it was just an oversight this test did not have that line while e.g. the one above did.
without manually blurring/focusing the actions fire in wrong order:

without manual blur:
* `__togglebutton_click__`
* `__togglebutton_click__`
* `__togglebutton_click__`
* `__input_blur__`

with:
* `__togglebutton_click__`
* `__togglebutton_click__`
* `__input_blur__`
* `__togglebutton_click__`